src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -117,7 +117,8 @@ const ActionSidebar: FC<PropsWithChildren<ActionSidebarProps>> = ({
           border-gray-200
           rounded-l-lg
           shadow-default
-          z-[60]
+          z-[65]
+          md:z-[60]
```

Comment:
If possible, it would be good not to change z-index depending on the screen width

Reply:
If I remember it right, @CzarekDryl was working on it in a separate PR. The solution I added is temporary, only to make it possible to see the changes on mobile right now. I will remove it when Czarek's changes will be merged.